diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1998679..ff2a31c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@ find_package(Threads REQUIRED)
 
 # Use all standard-compliant optimizations
 set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -mcx16 -g")
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -mcx16 -g")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -mcx16 -g -D_GLIBCXX_USE_C99_STDLIB")
 
 # Set the output folder where your program will be created
 set(CMAKE_BINARY_DIR ${CMAKE_SOURCE_DIR}/bin)
@@ -26,16 +26,16 @@ include_directories("${PROJECT_SOURCE_DIR}")
 include(${CMAKE_ROOT}/Modules/ExternalProject.cmake)
 
 # sdsl-lite (full build using its cmake config)
-ExternalProject_Add(sdsl-lite
-  SOURCE_DIR "${CMAKE_SOURCE_DIR}/deps/sdsl-lite"
-  CMAKE_ARGS "${CMAKE_ARGS};-DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>"
-  UPDATE_COMMAND ""
-  INSTALL_COMMAND "")
-ExternalProject_Get_property(sdsl-lite INSTALL_DIR)
-set(sdsl-lite_INCLUDE "${INSTALL_DIR}/src/sdsl-lite-build/include")
-set(sdsl-lite-divsufsort_INCLUDE "${INSTALL_DIR}/src/sdsl-lite-build/external/libdivsufsort/include")
-set(sdsl-lite_LIB "${INSTALL_DIR}/src/sdsl-lite-build/lib")
-set(sdsl-lite-divsufsort_LIB "${INSTALL_DIR}/src/sdsl-lite-build/external/libdivsufsort/lib")
+# ExternalProject_Add(sdsl-lite
+#  SOURCE_DIR "${CMAKE_SOURCE_DIR}/deps/sdsl-lite"
+#  CMAKE_ARGS "${CMAKE_ARGS};-DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>"
+#  UPDATE_COMMAND ""
+#  INSTALL_COMMAND "")
+#ExternalProject_Get_property(sdsl-lite INSTALL_DIR)
+#set(sdsl-lite_INCLUDE "${INSTALL_DIR}/src/sdsl-lite-build/include")
+#set(sdsl-lite-divsufsort_INCLUDE "${INSTALL_DIR}/src/sdsl-lite-build/external/libdivsufsort/include")
+#set(sdsl-lite_LIB "${INSTALL_DIR}/src/sdsl-lite-build/lib")
+#set(sdsl-lite-divsufsort_LIB "${INSTALL_DIR}/src/sdsl-lite-build/external/libdivsufsort/lib")
 
 # taywee's C++ args library, header only
 ExternalProject_Add(tayweeargs
@@ -166,7 +166,6 @@ add_executable(seqwish
   ${CMAKE_SOURCE_DIR}/src/mmap.cpp
   )
 add_dependencies(seqwish tayweeargs)
-add_dependencies(seqwish sdsl-lite)
 add_dependencies(seqwish gzipreader)
 add_dependencies(seqwish mmmulti)
 add_dependencies(seqwish iitii)
@@ -178,8 +177,8 @@ add_dependencies(seqwish atomicqueue)
 add_dependencies(seqwish ska)
 add_dependencies(seqwish paryfor)
 target_include_directories(seqwish PUBLIC
-  "${sdsl-lite_INCLUDE}"
-  "${sdsl-lite-divsufsort_INCLUDE}"
+  # "${sdsl-lite_INCLUDE}"
+  # "${sdsl-lite-divsufsort_INCLUDE}"
   "${tayweeargs_INCLUDE}"
   "${gzipreader_INCLUDE}"
   "${ips4o_INCLUDE}"
