diff --git a/CMakeLists.txt b/CMakeLists.txt
index 288833f..6e1a449 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,18 +117,15 @@ endif(BUILD_DYNAMIC_EXECUTABLE)
 
 
 
-# Install to the shasta-install directory.
-set(CMAKE_INSTALL_PREFIX .)
-
 # Install the scripts.
 file(GLOB SCRIPTS scripts/*.py scripts/*.sh)
-install(PROGRAMS ${SCRIPTS} DESTINATION shasta-install/bin)
+install(PROGRAMS ${SCRIPTS} DESTINATION share/shasta/scripts)
 
 # Install the configuration files.
-install(DIRECTORY conf DESTINATION shasta-install USE_SOURCE_PERMISSIONS)
+install(DIRECTORY conf DESTINATION share/shasta USE_SOURCE_PERMISSIONS)
 
 # Install the docs directory.
-install(DIRECTORY docs DESTINATION shasta-install)
+install(DIRECTORY docs DESTINATION share/shasta)
 
 # The targets built in each subdirectory are
 # installed by the cmake file of each subdirectory.
diff --git a/dynamicExecutable/CMakeLists.txt b/dynamicExecutable/CMakeLists.txt
index f3ecb18..0afb7dd 100644
--- a/dynamicExecutable/CMakeLists.txt
+++ b/dynamicExecutable/CMakeLists.txt
@@ -80,10 +80,10 @@ if(BUILD_DEBUG)
 endif(BUILD_DEBUG)
 
 # Install the dynamic executable into the bin directory.
-install(TARGETS shastaDynamicExecutable DESTINATION shasta-install/bin)
+install(TARGETS shastaDynamicExecutable DESTINATION bin)
 
 # When install is complete, create the AppImage.
 if(BUILD_APPIMAGE)
-install(CODE "execute_process(COMMAND ${CMAKE_SOURCE_DIR}/AppImage/CreateAppImage.py shasta-install)")
+install(CODE "execute_process(COMMAND ${CMAKE_SOURCE_DIR}/AppImage/CreateAppImage.py share/shasta/appimage)")
 endif(BUILD_APPIMAGE)
 
diff --git a/dynamicLibrary/CMakeLists.txt b/dynamicLibrary/CMakeLists.txt
index c143d07..fc0b141 100644
--- a/dynamicLibrary/CMakeLists.txt
+++ b/dynamicLibrary/CMakeLists.txt
@@ -88,8 +88,8 @@ target_link_libraries(
     shastaDynamicLibrary 
      atomic png boost_program_options pthread z spoa ${SHASTA_PYTHON_LIBRARIES})
 
-# Install the shared library into the bin directory.
-install(TARGETS shastaDynamicLibrary DESTINATION shasta-install/bin)
+# Install the shared library into the lib directory.
+install(TARGETS shastaDynamicLibrary DESTINATION lib)
 
 
 
diff --git a/staticExecutable/CMakeLists.txt b/staticExecutable/CMakeLists.txt
index e811e82..46c0653 100644
--- a/staticExecutable/CMakeLists.txt
+++ b/staticExecutable/CMakeLists.txt
@@ -101,7 +101,7 @@ if(NOT MACOS)
     SET(CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} -static-libstdc++ -static-libgcc")
 endif(NOT MACOS)
 
-# The static executable goes to the bin directory.
-install(TARGETS shastaStaticExecutable DESTINATION shasta-install/bin)
+# The static executable goes to the lib directory.
+install(TARGETS shastaStaticExecutable DESTINATION bin)
 
 
diff --git a/staticLibrary/CMakeLists.txt b/staticLibrary/CMakeLists.txt
index 004bc3e..e88389b 100644
--- a/staticLibrary/CMakeLists.txt
+++ b/staticLibrary/CMakeLists.txt
@@ -66,8 +66,8 @@ set_target_properties(shastaStaticLibrary PROPERTIES OUTPUT_NAME "shasta")
 set_target_properties(shastaStaticLibrary PROPERTIES PREFIX "")
 set_target_properties(shastaStaticLibrary PROPERTIES DEFINE_SYMBOL "")
 
-# Install the static library into the bin directory.
-install(TARGETS shastaStaticLibrary DESTINATION shasta-install/bin)
+# Install the static library into the lib directory.
+install(TARGETS shastaStaticLibrary DESTINATION lib)
 
 
 
